ts/features/login/preferences/components/LoginExpirationBanner.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Banner,
+  IOVisualCostants,
+  useIOToast
+} from "@pagopa/io-app-design-system";
+import { useCallback } from "react";
+import { StyleSheet, View } from "react-native";
+import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import { openWebUrl } from "../../../../utils/url";
+import I18n from "../../../../i18n";
+import { closeSessionExpirationBanner } from "../store/actions";
+import { formattedExpirationDateSelector } from "../../../../store/reducers/authentication";
+
+const SESSION_EXPIRED_EDUCATIONAL_URL =
+  "https://assistenza.ioapp.it/hc/it/articles/32616176301713-Cosa-fare-quando-la-sessione-scade";
+
+type Props = {
+  handleOnClose: () => void;
+};
+/**
+ * to use in case the banner's visibility has to be handled externally
+ * (see MultiBanner feature for the landing screen)
+ */
+export const LoginExpirationBanner = ({ handleOnClose }: Props) => {
+  const expirationDate = useIOSelector(formattedExpirationDateSelector);
+  const { error } = useIOToast();
+  const dispatch = useIODispatch();
+
+  const handleOnPress = () => {
+    openWebUrl(SESSION_EXPIRED_EDUCATIONAL_URL, () => {
+      error(I18n.t("global.jserror.title"));
+    });
+  };
```

Comment:
Don't forget to track the event with `trackHelpCenterCtaTapped` ([here](https://github.com/pagopa/io-app/blob/73f2d47f8d95ab0d4fab5b3ce06fbacdfd98f2c6/ts/utils/analytics.ts#L216C17-L216C41))